os/board/common/crashdump.c

```diff
@@ -208,6 +210,68 @@ static int get_ext_userfs_part(size_t *size, int *index)
 	return 0;
 }
 
+static int send_file(void)
+{
+	int i;
+	int fd;
+	int ret;
+	struct stat st;
+	off_t size;
+	char buf;
+	char filename[CONFIG_SMARTFS_MAXNAMLEN + 1];
+	char char_filesize;
+
+	/* Send acknowledgement for USERFS FILE DUMP Handshake */
+	up_lowputc('A');
+
+	/* Receive the file name to be sent to the host */
+	for (i = 0; i < CONFIG_SMARTFS_MAXNAMLEN + 1; i++) {
+		filename[i] = '\0';
+	}
```

Comment:
Is this necessary?
If necessary, how about doing at definition?
```
char filename[CONFIG_... + 1] = { 0, };
```

Reply:
Agreed